modules/lang-painless/src/test/java/org/elasticsearch/painless/node/NodeToStringTests.java

```diff
@@ -19,39 +19,16 @@
 
 package org.elasticsearch.painless.node;
 
-import org.elasticsearch.painless.CompilerSettings;
-import org.elasticsearch.painless.FeatureTestObject;
-import org.elasticsearch.painless.Scope.Variable;
-import org.elasticsearch.painless.Location;
-import org.elasticsearch.painless.Operation;
-import org.elasticsearch.painless.ScriptClassInfo;
-import org.elasticsearch.painless.action.PainlessExecuteAction.PainlessTestScript;
-import org.elasticsearch.painless.antlr.Walker;
-import org.elasticsearch.painless.lookup.PainlessCast;
-import org.elasticsearch.painless.lookup.PainlessClass;
-import org.elasticsearch.painless.lookup.PainlessField;
-import org.elasticsearch.painless.lookup.PainlessLookup;
-import org.elasticsearch.painless.lookup.PainlessLookupBuilder;
-import org.elasticsearch.painless.lookup.PainlessLookupUtility;
-import org.elasticsearch.painless.lookup.PainlessMethod;
-import org.elasticsearch.painless.spi.Whitelist;
-import org.elasticsearch.painless.spi.WhitelistLoader;
 import org.elasticsearch.test.ESTestCase;
 
-import java.util.ArrayList;
-import java.util.Arrays;
-import java.util.List;
-import java.util.Map;
-
-import static java.util.Collections.emptyList;
-import static java.util.Collections.singletonList;
+// TODO: fix these tests to match new node structure
 
 /**
  * Tests {@link Object#toString} implementations on all extensions of {@link ANode}.
  */
 public class NodeToStringTests extends ESTestCase {
 
-    public void testEAssignment() {
+    /*public void testEAssignment() {
```

Comment:
?

Reply:
Can you do `Test.skip("ignoring rapidly changing tests until quiescent, see issue #XXXX")`

Reply:
Done:
https://github.com/elastic/elasticsearch/issues/51842